ctapipe_io_lst/calibration.py

```diff
@@ -88,8 +88,8 @@ class LSTR0Corrections(TelescopeComponent):
     offset = IntTelescopeParameter(
         default_value=0,
         help=(
-            'Define the offset the offset of the baseline to be subtracted'
-            ' from the waveform *additionally* to the drs4 pedestal offset.'
+            'Define offset to be subtracted from the waveform *additionally*'
+            ' to the drs4 pedestal offset.'
             ' Only set this to a value != 0 when not using the drs4 offset '
```

Comment:
Is this true? We could want to subtract an additional offset also applying the drs4 correction offset. 
Otherwise, perhaps better to verify the apply_drs4_pedestal_correction flag before applying the offset and in the comment replace  *additionally* with *instead of*  (or similar)

Reply:
I don't understand what you mean.

The default no is 0, so no offset is subtracted. We now also do not subtract an offset from the drs4 pedestal coefficents.

That means for the "normal" case of supplying a `drs4_pedestal_file`, everything should be correct, or not?

Reply:
I refer to the phrase 'Only set this to a value != 0 when not using the drs4 offset'

Reply:
Yes, I know, that's where you added the comment.

Reply:
I mean, if we say that it is *additionally* to the drs4 pedestal offset, then it is not coherent  to ask to use it only if this is  not used

Reply:
Yes it is. Because normally, the offset is included in the drs4 pedestal and you only need an `offset !=0` when not using the drs4_offset or when the offset of the drs4 run is different than the data run.

Reply:
Hi @maxnoe, I continue to find the phrase "Only set this to a value != 0 when not using the drs4 offset " rather cryptical for a standard user and not necessary because the specification of *additionally* is enough to decide if to use or not this further offset

Reply:
Is that cleared now?